ui/push-health-view/Health.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table, Container } from 'reactstrap';
+import Icon from 'react-fontawesome';
```

Comment:
The fontawesome project renamed a bunch of their packages when creating Font Awesome v5. The rest of Treeherder is still using Font Awesome v4 (and so the old package names), but I think the new packages might work fine alongside them (so long as it's for different pages/entrypoints in the site). 

As such, perhaps we should try adding `@fortawesome/react-fontawesome` to `package.json` and using it here instead? The usage has changed slightly, see:
https://fontawesome.com/how-to-use/on-the-web/using-with/react
https://github.com/FortAwesome/react-fontawesome#introduction

We can then switch over the rest of Treeherder in the future in [bug 1506805](https://bugzilla.mozilla.org/show_bug.cgi?id=1506805) and deps :-)

Reply:
Very cool, yeah.  I switched the shared ``Login.jsx`` to use these, too.  That way this view doesn't have to import as much in ``index.jsx`` and the others are that much closer to being on FA-5.

Reply:
Awesome! I like new style of imports with Font Awesome 5 - seems much cleaner than passing strings via the `name` prop, and also allows for tree shaking :-)